### DISCUSSION TITLE
Fastapi container stops all of a sudden

### DISCUSSION THREAD

**Discussion by asleep-henry** [fastapi/fastapi#9247] [Questions]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI()

@app.get("/")
def read_root():
	return {"Hello": "World"}
```

### Description

I'm running a Fastapi server with container orchestration, sometimes a container goes down randomly.
The RSS memory of an Nginx container rises slightly and fails to process all requests.
I can't get any context of why this is happening.
![스크린샷 2023-03-12 오후 10 37 41](https://user-images.githubusercontent.com/112384442/224548597-5b11db9f-8cc1-4135-ae22-6cc58e2daab7.png)

### Operating System

Linux

### Operating System Details

linux/x86

### FastAPI Version

0.92.0

### Python Version

3.8.10

### Additional Context

It first happened at version 0.88.0.
I upgraded it to 0.92.0 but it still happens.

**Comment by meirarmon** [comment]
Can you provide more details about your orchestration? Dockerfile? Nginx config if applicable?

**Comment by acidjunk** [comment]
I wonder; what was done to resolve it.
